### ISSUE TITLE
Any updates?

### ISSUE THREAD

**Issue by zmnv** [jongold/figma-js#10]
Are you still here? 
Doesn't your type know about new Figma Api scheme?

![image](https://user-images.githubusercontent.com/25253501/51848103-8d359b80-234f-11e9-8af6-b7f83b504b00.png)

![image](https://user-images.githubusercontent.com/25253501/51848212-c1a95780-234f-11e9-9abc-ff15ed9c6620.png)

**Comment by zmnv** [comment]
```ts
export interface Style {
  readonly key: string;
  readonly name: string;
  readonly styleType: string;
}
```

**Comment by jem-computer** [comment]
Feel free to submit a pull request!

**Comment by zmnv** [comment]
Sounds interesting:) So, it's a true what schematic of api results has changed?
What do you think, they change it again soon?

**Comment by thibmaek** [comment]
Should this issue still be open after #11 was merged?

**Comment by chrisdrackett** [comment]
looks like this should be closed, feel free to report a new issue if I'm wrong!
